libexec/bats-format-tap-stream

```diff
@@ -51,6 +53,22 @@ skip() {
   advance
 }
 
+todo() {
+  local reason="$1"
+  [ -z "$reason" ] || reason=": $reason"
```

Comment:
Use `[[` and `]]`, and expand this into an `if` statement to make it easier to scan.